bot/cogs/fun.py

```diff
@@ -199,14 +199,54 @@ async def lizard(self, ctx) -> None:
                 else:
                     await ctx.send(f"Something went Boom! [status : {lizr.status}]")
 
+    @command(aliases=["leet"])
+    async def leetify(self, ctx: Context, *, content: str) -> None:
+        """Gives each letter of a given message a different markdown style."""
+
+        leetters = {
+            "a": ["A", "4"],
+            "e": ["E", "3"],
+            "i": ["1", "I", "i"],
+            "o": ["O", "0", "o"],
+            "s": ["5", "S", "s"]
+        }
+        content = content.lower()
+        leetified_content = ""
+        prev_md = ""
+        for letter in content:
+            if letter in leetters.keys():
+                leet_char = random.choice(leetters[letter])
+            else:
+                leet_char = random.choice([letter, letter.upper()])
+
+            # Apply markdown without using the same style next to each other
+            md_list = ["*", "**", "***", "__", "", "`"]
+            if prev_md == "":
+                md_list.remove("")
+            elif "*" in prev_md:
+                md_list.remove("*")
+                md_list.remove("**")
+                md_list.remove("***")
+            elif prev_md == "__":
+                md_list.remove("__")
+            elif prev_md == "`":
+                md_list.remove("`")
+            random.seed(random.randint(421, 294244))
+            chosen_md = random.choice(md_list)
+            prev_md = chosen_md
+            leetified_content += f"{chosen_md}{leet_char}{chosen_md}"
+
+        await ctx.send(f"{leetified_content}\n-{ctx.message.author.mention}")
+
     @command()
     async def why(self, ctx: Context) -> None:
         """Why?."""
         async with aiohttp.ClientSession() as session:
             async with session.get("https://nekos.life/api/why", headers=self.user_agent) as why:
                 if why.status == 200:
                     why_js = await why.json()
-                    why_em = discord.Embed(title=f"{ctx.author.name} wonders...", description=why_js["why"], colour=0x690E8,)
+                    why_em = discord.Embed(
+                        title=f"{ctx.author.name} wonders...", description=why_js["why"], colour=0x690E8,)
```

Comment:
We should use this formatting when splitting an embed, it is more readable.
We also don't need to use `discord.Embed` as we have `Embed` imported directly
```suggestion
                    why_em = Embed(
                        title=f"{ctx.author.name} wonders...", 
                        description=why_js["why"], 
                        colour=0x690E8
                    )
```

Reply:
Hey, you didn't actually resolve this one, please avoid marking unresolved requests as resolved.

Reply:
I've gone through the embeds, is there any particular reason for a trailing comma at the embed constructors arguments end?
`Embed(title="title", color=color.gold(),)`